Esmorga-Backend/esmorga-backend
# [MOB-130] BE - POST Email verification

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-130)**

## Testing Guide
After making a registration request and without verifying the account call this request and see how the verificationCode is overwritten in the database and the activation email is resend:
```
curl --location 'http://localhost:3000/v1/account/email/verification' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "test-email-verification@yopmail.com"
}'
```
